SocialCareCaseViewerApi/Properties/launchSettings.json

```diff
@@ -1,5 +1,4 @@
 ﻿{
-  "$schema": "http://json.schemastore.org/launchsettings.json",
```

Comment:
launch settings are usually set by each developer to their liking, so no need to commit this file at all